src/Layers/xrRender/Blender_tree.cpp

```diff
@@ -202,103 +202,109 @@ void CBlender_Tree::Compile(CBlender_Compile& C)
     switch (C.iElement)
     {
     case SE_R2_NORMAL_HQ: // deffer
-        uber_deffer(C, true, tvs, "base", oBlend.value, nullptr, false, "clr");
+        uber_deffer(C, true, tvs, "base", oBlend.value, nullptr, false, C._defSubPath);
         break;
     case SE_R2_NORMAL_LQ: // deffer
-        uber_deffer(C, false, tvs, "base", oBlend.value, nullptr, false, "clr");
+        uber_deffer(C, false, tvs, "base", oBlend.value, nullptr, false, C._defSubPath);
         break;
     case SE_R2_SHADOW: // smap-spot
         //	TODO: DX10: Use dumb shader for shadowmap since shadows are drawn using hardware PCF
         if (oBlend.value)
             C.r_Pass(tvs_s, "shadow_direct_base_aref", FALSE, TRUE, TRUE, TRUE, D3DBLEND_ZERO, D3DBLEND_ONE, TRUE, 200);
         else
             C.r_Pass(tvs_s, "shadow_direct_base", FALSE);
-        //C.r_Constant("recolor_s", &binder_recolor_tex);
+        // C.r_Constant("recolor_s", &binder_recolor_tex);
         C.r_Sampler("s_base", C.L_textures[0]);
         C.r_End();
         break;
     }
 }
-#elif RENDER==R_GL
+#elif RENDER == R_GL
 //////////////////////////////////////////////////////////////////////////
 // GL
 //////////////////////////////////////////////////////////////////////////
 #include "uber_deffer.h"
-void	CBlender_Tree::Compile	(CBlender_Compile& C)
+void CBlender_Tree::Compile(CBlender_Compile& C)
 {
-	IBlender::Compile	(C);
+    IBlender::Compile(C);
 
-	//*************** codepath is the same, only shaders differ
-	LPCSTR	tvs;
-	LPCSTR	tvs_s;
-	if (oNotAnTree.value)	
-	{ 
-		tvs="tree_s";
-		if (oBlend.value)	tvs_s="shadow_direct_tree_s_aref"; 
-		else	tvs_s="shadow_direct_tree_s"; 
-	}
-	else
-	{
-		tvs				= "tree";
-		if (oBlend.value)	tvs_s="shadow_direct_tree_aref"; 
-		else	tvs_s="shadow_direct_tree"; 
-	}
+    //*************** codepath is the same, only shaders differ
+    LPCSTR tvs;
+    LPCSTR tvs_s;
+    if (oNotAnTree.value)
+    {
+        tvs = "tree_s";
+        if (oBlend.value)
+            tvs_s = "shadow_direct_tree_s_aref";
+        else
+            tvs_s = "shadow_direct_tree_s";
+    }
+    else
+    {
+        tvs = "tree";
+        if (oBlend.value)
+            tvs_s = "shadow_direct_tree_aref";
+        else
+            tvs_s = "shadow_direct_tree";
+    }
 
-	bool bUseATOC = (oBlend.value && (RImplementation.o.dx10_msaa_alphatest==CRender::MSAA_ATEST_DX10_0_ATOC));
+    bool bUseATOC = (oBlend.value && (RImplementation.o.dx10_msaa_alphatest == CRender::MSAA_ATEST_DX10_0_ATOC));
 
-	switch (C.iElement)
-	{
-	case SE_R2_NORMAL_HQ:	// deffer
-		if (bUseATOC)
-		{
-			uber_deffer		(C,true,tvs,"base_atoc",oBlend.value,0,true);
-			C.r_Stencil		( TRUE,D3DCMP_ALWAYS,0xff,0x7f,D3DSTENCILOP_KEEP,D3DSTENCILOP_REPLACE,D3DSTENCILOP_KEEP);
-			C.r_ColorWriteEnable(false, false, false, false);
-			C.r_StencilRef	(0x01);
-			//	Alpha to coverage.
-			C.RS.SetRS	(XRDX10RS_ALPHATOCOVERAGE,	TRUE);
-			C.r_End			();
-		}
+    switch (C.iElement)
```

Comment:
Please move 'styling' changes to a separate commit to keep the history clean. A mix of style corrections and real changes makes it harder to distinguish the meaningful moments. Nothing changed since #401.